python-sdk/example_dags/example_transform_sql.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
```

Comment:
This example DAG won't be run with our test suite unless we add it to:
https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_example_dags.py#L50

Perhaps we should log a separate ticket to run all the example DAGs as part of that test (as opposed to have to manually add them).

Reply:
Thanks 👍 I will add it to the test suite.

Reply:
> Perhaps we should log a separate ticket to run all the example DAGs as part of that test (as opposed to have to manually add them).

Not sure, because for some example dags it might require a lot of infra setup and imo example dags are for users to see how it works. It does not necessarily mean we are running them in our CI. Of course, it would be ideal if we test all our example dags. But I think this additional step makes sense to not having to block adding examples dags until we have tests setup.

Reply:
We also need to figure out costs. Not every DAG can be executed as often as others.

Reply:
Let's loop in more people to this discussion:
@kaxil @utkarsharma2 @phanikumv 

In the past we had issues with having DAGs which were no longer valid, and we had decided to run all of them using the `test_example_dags.py`. At the moment we are very likely missing DAGs and not running them in the suite. What are your thoughts?

Reply:
Yes, I would for the example DAGs to be run in the CI -- to make it simple you could use SQLite

Reply:
and looks like this is using sqlite :) so should be easy enough to test

Reply:
Alright, I can add another PR for running all examples dags - instead of having to add them manually.

Reply:
PR https://github.com/astronomer/astro-sdk/pull/992 for running all examples dags.